Sources/Data/models/Model.xcdatamodeld/Model17.xcdatamodel/contents

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="21512" systemVersion="22A400" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="21513" systemVersion="21G115" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
```

Comment:
Have we shipped model 17 yet? Does this need a new model?

Reply:
This will actually work properly with the same model since we removed the models and relations for it but it is a better practice always to create a new one after removals. But this made me realize I forgot to remove a model and a relation with it. Thanks for the warning.

Fix at https://github.com/brave/brave-ios/pull/7087/commits/d7a9e69bec7e07a2d47694715b8c1a0a24fe1178